commit 1a8db64ba37583ef452f7fc2b4fade8daf5b2eea
Author: Matt Weeks <cityismychurch@users.noreply.github.com>
Date:   Sat Feb 25 13:45:32 2017 -0500

    Borderless windows

diff --git a/sources/PseudoTerminal.m b/sources/PseudoTerminal.m
index f3e4d2e8..82a08c2f 100644
--- a/sources/PseudoTerminal.m
+++ b/sources/PseudoTerminal.m
@@ -412,7 +412,6 @@ static NSRect iTermRectCenteredVerticallyWithinRect(NSRect frameToCenter, NSRect
 
         default:
             return (mask |
-                    NSTitledWindowMask |
                     NSClosableWindowMask |
                     NSMiniaturizableWindowMask |
                     NSResizableWindowMask |
@@ -858,7 +857,7 @@ ITERM_WEAKLY_REFERENCEABLE
     // if we're in fullscreen.
     return ([iTermPreferences boolForKey:kPreferenceKeyEnableDivisionView] &&
             !togglingFullScreen_ &&
-            (self.window.styleMask & NSTitledWindowMask) &&
+            (self.window.styleMask) &&
             ![self anyFullScreen] &&
             ![self tabBarVisibleOnTop]);
 }
