diff --git a/sources/PseudoTerminal.m b/sources/PseudoTerminal.m
index 11b939ad2..6817f011c 100644
--- a/sources/PseudoTerminal.m
+++ b/sources/PseudoTerminal.m
@@ -507,7 +507,6 @@ static BOOL iTermWindowTypeIsCompact(iTermWindowType windowType) {
             if (@available(macOS 10.14, *)) {
                 return (mask |
                         NSWindowStyleMaskFullSizeContentView |
-                        NSWindowStyleMaskTitled |
                         NSWindowStyleMaskClosable |
                         NSWindowStyleMaskMiniaturizable |
                         NSWindowStyleMaskResizable);
@@ -525,7 +524,6 @@ static BOOL iTermWindowTypeIsCompact(iTermWindowType windowType) {
         case WINDOW_TYPE_COMPACT:
             return (mask |
                     NSWindowStyleMaskFullSizeContentView |
-                    NSWindowStyleMaskTitled |
                     NSWindowStyleMaskClosable |
                     NSWindowStyleMaskMiniaturizable |
                     NSWindowStyleMaskResizable);
@@ -533,14 +531,12 @@ static BOOL iTermWindowTypeIsCompact(iTermWindowType windowType) {
         case WINDOW_TYPE_COMPACT_MAXIMIZED:
             return (mask |
                     NSWindowStyleMaskFullSizeContentView |
-                    NSWindowStyleMaskTitled |
                     NSWindowStyleMaskClosable |
                     NSWindowStyleMaskMiniaturizable |
                     NSWindowStyleMaskTexturedBackground);
 
         case WINDOW_TYPE_MAXIMIZED:
             return (mask |
-                    NSWindowStyleMaskTitled |
                     NSWindowStyleMaskClosable |
                     NSWindowStyleMaskMiniaturizable |
                     NSWindowStyleMaskTexturedBackground);
@@ -554,7 +550,6 @@ static BOOL iTermWindowTypeIsCompact(iTermWindowType windowType) {
                 }
             }
             return (mask |
-                    NSWindowStyleMaskTitled |
                     NSWindowStyleMaskClosable |
                     NSWindowStyleMaskMiniaturizable |
                     NSWindowStyleMaskResizable |
